qds_sdk/cluster.py

```diff
@@ -286,7 +330,9 @@ def _parse_create_update(cls, args, action):
                                  default=None,
                                  help="don't encrypt the ephemeral drives on" +
                                       " the instance",)
-        security_group.add_argument("--customer-ssh-key",
+
+        customer_ssh_key_parameter = "--ssh-public-key" if (api_version >= 1.3) else "--customer-ssh-key"
```

Comment:
We have changed the command line flag in the new api version from "customer_ssh_key" to "ssh_public_key". I think its better to change this flag also when someone is using >= 1.3.